services/search/api.py

```diff
@@ -206,6 +237,11 @@ def get(self, request):
         if order_units_by_num_services:
             units_order_list.append("-num_services")
 
+        if "include" in params:
+            include_fields = params["include"].split(",")
+            # breakpoint()
```

Comment:
Remove this line

Reply:
> The first commit message is a bit misleading as it is dealing only with Swagger-documentation update and the same commit message is also used twice within this PR. Try to change the first commit message to be more descriptive. Otherwise looks good.

Fixed

Reply:
> Remove this line

Fixed